alignit/robots/xarm.py

```diff
@@ -33,10 +33,12 @@ def send_action(self, action):
         self.robot.send_action(action)
 
     def get_observation(self):
-        rgb_image = self.camera.read()
+        rgb_image,depth_image,acqusition_time = self.camera.async_read()
```

Comment:
```suggestion
        rgb_image, depth_image, acquisition_time = self.camera.async_read()
```
Space after comma, `acquisition` misspelled